internal/sops/keyservice/server.go

```diff
@@ -109,6 +256,18 @@ func (ks Server) Encrypt(ctx context.Context,
 		response = &keyservice.EncryptResponse{
 			Ciphertext: ciphertext,
 		}
+	case *keyservice.Key_AzureKeyvaultKey:
+		if _, err := os.Stat(filepath.Join(ks.HomeDir, "azure_kv.json")); os.IsNotExist(err) {
```

Comment:
The RootFS is readonly, there is no way `azure_kv.json` could be in the home dir. Please remove this code.

Reply:
ks.HomeDir is not the root fs but a temporary directory already used by the decryptor for gpg.